src/intents.cs

```diff
@@ -24,9 +24,11 @@
 #endif
 
 #if MONOMAC
+using AppKit;
 using UIImage = Foundation.NSObject;
 #else
 using UIKit;
+using NSImage = Foundation.NSObject;
```

Comment:
Kinda ugly, I know, and you might think, why not:
```csharp
using UIImage = AppKit.NSImage;
```
But there is a 🕷️ sense that tells me it would kick our asses in the future.

Reply:
anyway the selector would be named differently than `imageWithNSImage:`

Reply:
@spouliot  do we trust apple?

Reply:
only ~as~ half far as I can throw them